### DISCUSSION TITLE
Allow ![image-name](https://location.of.image/image.png) or <img /> into emails

### DISCUSSION THREAD

**Discussion by yvovandoorn** [pretix/pretix#3416] [Ideas]
I wanted to send out an email to attendees and render a location map only to be foiled that Pretix actively doesn't render `![image-name](https://location.of.image/image.png)` or `<img />`. (Even Github renders the markdown 🥲)

While that might be a way to send out 'bad' emails, I think it should be left to the organizer (especially when hosting on your own) whether they want enable this or not. Because now I have to use something like `listmonk` to send mails with an image to my attendees.

**Comment by eiclu** [comment]
To bump this, I would additionaly love to be allowed to use <img> tags in markdown fields. Are there security reasons why this isn't allowed right now?

**Reply by raphaelm** [reply]
Yeah, we do not allow external content in the page for security reasons, so it would require a way of hosting the image within pretix which makes it non-trivial. We also see very little use cases for images in markdown content that harm neither accessibility nor usability of the shop.
